### PR TITLE
Fixed tablesorter initialization for pages history and schedule.

### LINKED ISSUE #1942: Empty column selector in Schedule
@p0psicles 

no JS errors in F12

image

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

Fixes the tablesorter for the schedule and history page.
Schedule
* Fixed ajax ep search spinner.

History
* Fixed sorter in detailed view.

Fixes #1942

### CODE REVIEW

**Comment by fernandog** [comment]
approved

File: static/js/core.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ $.extend({
                 return m[1].toUpperCase();
             });
         }
-        return (reg).test(MEDUSA.config[pyVar]);
+        return (reg).test(MEDUSA.config.layout[pyVar]);
```
Comment (OmgImAlexis) [inline static/js/core.js]: This isn't correct. This method isn't just for layout values.

Comment (p0psicles) [inline static/js/core.js]: Ok I thought it was, as isMeta() was only used by the layouts. Then I suggest you fix it xo.

Comment (OmgImAlexis) [inline static/js/core.js]: I'll check if it is, if it's used else where then we need another fix. I'll look at this tomorrow morning. 👍

**Comment by p0psicles** [comment]
@OmgImAlexis what do you think of this? Passing it an object: {'layout': 'history'} you can check an additional level.

**Pull request merged**